src/utils/axios.js

```diff
@@ -2,10 +2,7 @@ import { useUserStore } from "@/store/userStore";
 import axios, { AxiosError } from "axios";
 
 const axiosInstance = axios.create({
-  baseURL:
-    process.env.NEXT_PUBLIC_ENVIRONMENT == "local"
-      ? process.env.NEXT_PUBLIC_API_BASE_URL_LOCAL
-      : process.env.NEXT_PUBLIC_API_BASE_URL,
+  baseURL: process.env.NEXT_PUBLIC_BACKEND_API_URL,
```

Comment:
The .env variable for the backend URL is `NEXT_PUBLIC_API_BASE_URL` on vercel. I will merge and change this line; please take note.